api/v1beta1/module_types.go

```diff
@@ -63,7 +63,7 @@ type Build struct {
 	Dockerfile string `json:"dockerfile"`
 
 	// +optional
-	// Pull contains settings determining how to check if the DriverContainer image already exists.
+	// Pull contains settings determining how to pull the base images of the build process.
 	Pull PullOptions `json:"pull"`
```

Comment:
Should that one be made a pointer?

Reply:
I see no reason for it right now. The other 2 are pointers since they can be overriden ( global or in mapping), but the one in Build is not overriden, and the default values are false